perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ResourcesManagerMethod.java

```diff
@@ -1375,5 +1375,47 @@ public Object call(ApiCaller ac, Deserializer parms) throws PerunException {
 
 			return null;
 		}
-	};
+	},
+
+	/*#
+	 * Lists all of the resource assignments for the given group. Also, returns specified attributes
+	 * for the resources. If attrNames are null or empty, all resource attributes are returned.
+	 *
+	 * @param group int Group <code>id</code>
+	 * @param attrNames List<String> names of attributes to return
+	 * @return List<AssignedResource> resources for given group with specified attributes
+	 */
```

Comment:
We also mention here which exact exceptions can be thrown by this call. These exceptions are taken from the method at the entry layer. Just the PrivilegedException doesn't need to be mentioned, it is added to all methods automatically.

Reply:
Added